base/applications/drwtsn32/main.cpp

```diff
@@ -143,49 +147,131 @@ int abort(FILE* output, int err)
     return err;
 }
 
-int main(int argc, char* argv[])
+std::wstring Settings_GetOutputPath(void)
 {
+    WCHAR Buffer[MAX_PATH + 1];
```

Comment:
It's easier to just do WCHAR Buffer[MAX_PATH + 1] = {0};

Reply:
Actually the value number of MAX_PATH already counts for the terminating NULL.
And, it's possible to do: `WCHAR Buffer[whatever] = L"";`